File: src/main/kotlin/com/zuhaib/FinalCaseAxis/service/Impl/CSVServiceImpl.java
package com.zuhaib.FinalCaseAxis.service.Impl;

import com.zuhaib.FinalCaseAxis.helper.CSVUtils;
import com.zuhaib.FinalCaseAxis.model.helper.RevokeRequestModel;
import org.springframework.stereotype.Service;
import org.springframework.web.multipart.MultipartFile;

import java.io.IOException;
import java.util.List;

@Service
public class CSVServiceImpl {

    private CSVUtils csvUtils;

    public List<RevokeRequestModel> uploadMultipart(MultipartFile file) throws IOException {
        var ot = csvUtils.read(RevokeRequestModel.class, file.getInputStream());
        System.out.println(ot);
        return ot;
    }

}

File: src/main/kotlin/com/zuhaib/FinalCaseAxis/service/Impl/CSVServiceImpl.kt
package com.zuhaib.FinalCaseAxis.service.Impl

import com.zuhaib.FinalCaseAxis.helper.CSVUtils
import com.zuhaib.FinalCaseAxis.model.helper.RevokeRequestModel
import kotlin.Throws
import java.io.IOException
import org.springframework.web.multipart.MultipartFile
import org.springframework.stereotype.Service

@Service
class CSVServiceImpl {
    private val csvUtils: CSVUtils? = null
    @Throws(IOException::class)
    fun uploadMultipart(file: MultipartFile): List<RevokeRequestModel> {
        val ot = CSVUtils.read(RevokeRequestModel::class.java, file.inputStream)
        println(ot)
        return ot
    }
}